test/integration/narrow/routes/monitoring.test.js

```diff
@@ -261,4 +262,61 @@ describe('monitoring test', () => {
     const response = await server.inject(options)
     expect(response.statusCode).toBe(404)
   })
+
+  test('GET /monitoring/view-processed-payment-requests route returns 403 if user not in role', async () => {
+    auth.credentials.scope = []
+    const options = {
+      method: 'GET',
+      url: '/monitoring/view-processed-payment-requests',
+      auth
+    }
+
+    const response = await server.inject(options)
+    expect(response.statusCode).toBe(403)
+  })
+
+  test('GET /monitoring/view-processed-payment-requests route redirects to login page if not authorised', async () => {
+    const options = {
+      method: 'GET',
+      url: '/monitoring/view-processed-payment-requests'
+    }
+
+    const response = await server.inject(options)
+    expect(response.statusCode).toBe(302)
+    expect(response.headers.location).toEqual('/login')
+  })
+
+  test('GET /monitoring/view-processed-payment-requests route returns 200 if V2 events enabled', async () => {
+    const options = {
+      method: 'GET',
+      url: '/monitoring/view-processed-payment-requests',
+      auth
+    }
+
+    const response = await server.inject(options)
+    expect(response.statusCode).toBe(200)
+  })
+
+  test('GET /monitoring/view-processed-payment-requests route returns view-processed-payment-requests view if V2 events enabled', async () => {
+    const options = {
+      method: 'GET',
+      url: '/monitoring/view-processed-payment-requests',
+      auth
+    }
+
+    const response = await server.inject(options)
+    expect(response.payload).toContain('processed payment requests')
+  })
+
+  test('GET /monitoring/view-processed-payment-requests route returns 404 if V2 events disabled', async () => {
```

Comment:
Does it only return 404 or does it also route to another view when V2 events disabled?

Reply:
i think threre is 404 page that it returns, i followed the format of the tests above for these ones.